src/gui/virtualstudio.cpp

```diff
@@ -290,6 +290,26 @@ QString VirtualStudio::connectionState()
     return m_connectionState;
 }
 
+bool VirtualStudio::showInactive()
+{
+    return m_showInactive;
+}
+
+void VirtualStudio::setShowInactive(bool val)
+{
+    m_showInactive = val;
```

Comment:
For consistency with the rest of the code, can we use inactive rather than val here? (Same below.)